phac-nml/irida-next
# New project subgroup/searchable dropdown

## What does this PR do and why?
_Describe in detail what your merge request does and why._
This PR adds the searchable dropdown to the new Project and new Subgroup forms. This dropdown will make it easier for a user to look up a specific Namespace or Group when creating a new Project or Subgroup.

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._
![image](https://github.com/user-attachments/assets/439fdea0-e461-4b87-a1b2-7e2b31868f43)
![image](https://github.com/user-attachments/assets/cea69dfc-dee8-438a-b45f-8cf411487e89)


## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._
1. Login
2. In the sidebar, click on Groups
3. Select a group that your are an owner of
4. On the details page, click on the New project button
5. In the Project URL, search for any Namespaces that you have at least the Maintainer role for and check that they present on the list.
6. Repeat steps 1-5 with the New subgroup button and Group URL section.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
